notebooks/signac_204_External_Tools.ipynb

```diff
@@ -6,16 +6,47 @@
    "source": [
```

Comment:
<p>Remove this one as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>